index.js

```diff
@@ -21,6 +22,105 @@ module.exports = (app, { getRouter }) => {
     })
   }
 
+  app.on(
+    [
+      'pull_request.opened',
+      'pull_request.reopened',
+      'pull_request.synchronize',
+    ],
```

Comment:
Have you tested that this also works for GitHub actions?

Reply:
Not yet

Reply:
Would you mind testing it? 😅

Reply:
Will do.

Reply:
Seems to working with my initial tests. I updated the README howto enable.